model/testresult/testresult.go

```diff
@@ -29,19 +29,32 @@ type TaskTestResultsStats struct {
 
 // TestResult represents a single test result from an Evergreen task run.
 type TestResult struct {
-	TaskID          string    `json:"task_id" bson:"task_id"`
-	Execution       int       `json:"execution" bson:"execution"`
-	TestName        string    `json:"test_name" bson:"test_name"`
-	DisplayTestName string    `json:"display_test_name" bson:"display_test_name"`
-	GroupID         string    `json:"group_id" bson:"group_id"`
-	Status          string    `json:"status" bson:"status"`
-	BaseStatus      string    `json:"base_status" bson:"base_status"`
-	LogTestName     string    `json:"log_test_name" bson:"log_test_name"`
-	LogURL          string    `json:"log_url" bson:"log_url"`
-	RawLogURL       string    `json:"raw_log_url" bson:"raw_log_url"`
-	LineNum         int       `json:"line_num" bson:"line_num"`
-	TestStartTime   time.Time `json:"test_start_time" bson:"test_start_time"`
-	TestEndTime     time.Time `json:"test_end_time" bson:"test_end_time"`
+	TaskID          string       `json:"task_id" bson:"task_id"`
+	Execution       int          `json:"execution" bson:"execution"`
+	TestName        string       `json:"test_name" bson:"test_name"`
+	DisplayTestName string       `json:"display_test_name" bson:"display_test_name"`
+	GroupID         string       `json:"group_id" bson:"group_id"`
+	Status          string       `json:"status" bson:"status"`
+	BaseStatus      string       `json:"base_status" bson:"base_status"`
+	LogInfo         *TestLogInfo `json:"log_info" bson:"log_info"`
+	TestStartTime   time.Time    `json:"test_start_time" bson:"test_start_time"`
+	TestEndTime     time.Time    `json:"test_end_time" bson:"test_end_time"`
+
+	// Legacy test log fields.
+	LogTestName string `json:"log_test_name" bson:"log_test_name"`
+	LogURL      string `json:"log_url" bson:"log_url"`
+	RawLogURL   string `json:"raw_log_url" bson:"raw_log_url"`
+	LineNum     int    `json:"line_num" bson:"line_num"`
+}
+
+// TestLogInfo describes a metadata for a test result's log stored using
+// Evergreen logging.
+type TestLogInfo struct {
+	LogName       string    `json:"log_name" bson:"log_name"`
+	LogsToMerge   []*string `json:"logs_to_merge" bson:"logs_to_merge"`
+	LineNum       int32     `json:"line_num" bson:"line_num"`
+	RenderingType *string   `json:"rendering_type" bson:"rendering_type"`
```

Comment:
Should this be an enum, or is a string the best choice?

Reply:
It could be an enum, but the choices really depend on Parsley which feels external enough that I don't want to hardcode the rendering type options here.